systemverilog/SystemVerilog.g4

```diff
@@ -0,0 +1,3558 @@
+// Author: Mustafa Said Ağca
+// License: MIT
+
+grammar SystemVerilog;
```

Comment:
Please rename this grammar and file to `SystemVerilogParser` and lexer to `SystemVerilogLexer` because it's the common format for this repository.

```suggestion
grammar SystemVerilogParser;
```